app/policies/review_policy.rb

```diff
@@ -0,0 +1,30 @@
+class ReviewPolicy < ApplicationPolicy
```

Comment:
Not ReviewPolicy ! It's a BookingPolicy! 
Don't forget to change the file name to booking_policy.

Reply:
@RuiSMagalhaes updated